### PR TITLE
Fix English typos and misc details detected by PyCharm

### PR DESCRIPTION
**Pull request by mathbunnyru**
Sometimes I like to use PyCharm to write python/markdown code.
So I decided to fix some issues to not to see them anymore.

This PR doesn't fix all the issues, though, but I think the remaining ones are not actually issues and they have to do with some our internal names like `jovyan`.

### CODE REVIEW

File: docs/using/recipes.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ Sometimes it is helpful to run the Jupyter instance behind a nginx proxy, for ex
 
 - you would prefer to access the notebook at a server URL with a path
   (`https://example.com/jupyter`) rather than a port (`https://example.com:8888`)
-- you may have many different services in addition to Jupyter running on the same server, and want
+- you may have many services in addition to Jupyter running on the same server, and want
   to nginx to help improve server performance in managing the connections
```
Comment (consideRatio) [inline docs/using/recipes.md:197]:
```suggestion
  nginx to help improve server performance in managing the connections
```

File: tagging/taggers.py [hunk 2]
```diff
@@ -22,16 +22,18 @@ def _get_env_variable(container: Container, variable: str) -> str:
     raise KeyError(variable)
 
 
+_PIP_VERSION_PREFIX = "Version: "
```
Comment (consideRatio) [inline tagging/taggers.py:25]: I think this belongs within the function, it isn't something reused elsewhere anyhow right? If it's in this bigger scope, I'd assume it was.

**Review by consideRatio — APPROVED** [review]
I left some suggestions, if resolved - this LGTM!

I hope you don't mind me updating the PR description, I couldn't understand what "Fix PyCharm issues" meant. Initially I thought you had issues working with the code base with PyCharm.

**Comment by mathbunnyru** [comment]
Sure, feel free to update anything that is not clear :)

File: tagging/taggers.py (hunk 2 above)
Comment (mathbunnyru) [reply tagging/taggers.py:25]: Ok, I think the best thing to do is to just revert this.

**Pull request merged**